src/commands/chat/ui/UserMessageInput.tsx

```diff
@@ -21,9 +21,9 @@ export function UserMessageInput({ onSubmit }: UserMessageInputProps) {
   };
 
   return (
-    <Box flexDirection="row">
```

Comment:
Why change from Box to Text?

Reply:
iirc there was some issue with `Box` when you reside the terminal,  but i am not sure atm what exactly was it.